src/components/InventoryV2/Search.tsx

```diff
@@ -233,7 +233,9 @@ export default function Search(props: SearchProps): JSX.Element | null {
             setFilters={setFilters}
             label={strings.SPECIES}
             filterKey='speciesIds'
-            options={(species || []).map((n: Species) => n.id)}
+            options={[...(species || [])]
+              .sort((a, b) => a.scientificName.localeCompare(b.scientificName, activeLocale || undefined))
```

Comment:
Have you tested whether this is efficient for a long list, like the GBIF one?

Reply:
I have not, where can I find that list? Or is there a view in staging / vercel that would have that list?

Reply:
In admin, you can upload the list (it also has a link to download it) 
<img width="1035" alt="image" src="https://github.com/terraware/terraware-web/assets/114949086/fbb83c4e-5dc0-44c6-b12a-8be32ee53f02">

Also looks like Alex's Org is one example of a staging org with that list (https://tree-location-82tn94tte-terraformation.vercel.app/species?organizationId=28)

Reply:
I didn't notice any delay in there just now, so not a worry. Just wanted to ask since sorting by `localeCompare` is sometimes inefficient, I've read.